app/assets/stylesheets/application.css.scss

```diff
@@ -228,27 +231,31 @@ a.btn.btn-default:hover {
     font-size: 10px;
   }
 
-  .picture_background {
-    margin: -30px 5px 5px 5px;
-  }
-
   .photo-text {
     width: 100%;
     margin-left: -12px;
   }
+  
+  .picture_background {
+    margin-top: -30px;
```

Comment:
Properties should be ordered margin-bottom, margin-left, margin-right, margin-top